### DISCUSSION TITLE
Notion integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5660] [Feedback]
## Issue - Enhancement

I'd like to run meeting in Parabol but manage tasks in Notion. Right now, there's confusion about what goes where and some overlapping information. Much like we have for GitHub, I'd like to be able to add a task during a meeting in Parabol and send it to Notion. 

### Acceptance Criteria (optional)
Users can:
   - Send a task created in Parabol to Notion
   - See which tasks in Parabol are being managed in Notion

[a comment by an automated account is not shown]

**Comment by jordanh** [comment]
Notion lacks any public API. For now. I have a feeling this will reopen when this changes

**Comment by jordanh** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/3497801) writes: 🔒 

> any chance your team is working on an integration w/ notion? Here is my dream: We have leadership team meetings on Monday where everyone shares/aligns on their 2-3 big rocks for the week. I want to have a shared page on notion where these are tracked each week and people report on their progress/success at EOW. I then want to share this broadly (w/ full team, with investors, etc.).  Would love to push our cards from Parabol to notion for tracking/public sharing.

**Comment by jordanh** [comment]
Keeping this iceboxed until Notion releases there API

**Comment by avivapinchas** [comment]
Notion has an API 🎉 

I see a few use cases we could cover here: 
- As a team that uses Notion as our primary tasking interface, I want to create a task during a meeting in Parabol and publish it to Notion. <- I imagine this is useful for folks using our check-in meetings (we would probably use it!), but less useful for retros, because those types of tasks are different 
- As a team that uses Notion for our backlog, I want to pull in cards to Parabol and use the Sprint Poker meeting to define the value, effort of potential issues to determine what to work on next. I want Notion to be my source of truth for my backlog. <- this would help our marketing team _today_ 
- As a team that uses Notion for tasking, I want to remember what I worked and what comments I made over this sprint, to inspire reflections during a retro. <- for a later feature 

@jordanh Is this what you need to be able to determine the priority level for this item?

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Is this discussion still going on?

Recently, Notion has released Notion Project, which allows task management using native templates.

We are using Notion Project for task management and would like to use Parabol to estimate task points.

**Comment by AwesomeBobX64** [comment]
I'd like to use Parabol for Retros and output to Notion so we can keep track of our discussions and tasks/action items.
